modules/c8000v/variables.tf

```diff
@@ -0,0 +1,148 @@
+variable "metro_code" {
+  description = "Device location metro code"
+  type        = string
+  validation {
+    condition     = can(regex("^[A-Z]{2}$", var.metro_code))
+    error_message = "Valid metro code consists of two capital letters, i.e. SV, DC."
+  }
+}
+
+variable "project_id" {
+  description = "project_id"
+  type        = string
+}
+
+variable "account_number" {
+  description = "Billing account number for a device"
+  type        = string
+  default     = 0
```

Comment:
Can we add a similar validation rule as acl template for account number?

Reply:
done